infra/services/load_balancer.tf

```diff
@@ -90,6 +90,16 @@ resource "aws_security_group" "private_elb_security_group" {
     ]
   }
 
+  ingress {
+    from_port = "7799"
+    to_port = "7799"
+    protocol = "tcp"
+    cidr_blocks = [
+      "${data.terraform_remote_state.network.nat_public_ip}/32",
+      "${var.private_network_cidr}"
+    ]
+  }
+
   egress {
```

Comment:
This security group is for the web component rather than the worker. Only externally accessible services need a load balancer, hence there being one for web but not for worker.

The new port is on the worker. For private communications, we currently open all inbound ports to ECS so there's no need to update a security group. However, you do need to map the port on the host to the port in the worker container. This is done in infra/services/container-definitions/worker.json.tpl which is the template for the worker container definition.